changhyun/.env.sample

```
@@ -0,0 +1,9 @@
+DATABASE_URL = express_typeorm
+
+TYPEORM_CONNECTION = mysqul
+TYPEORM_HOST = host number
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = myPassword
+TYPEORM_DATABASE = express_typeorm
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING = TRUE
```

```suggestion
DATABASE_URL = "rdbmsType://username:password@iphost:portNumber/databaseName"

PORT = PORT

TYPEORM_CONNECTION = rdbms
TYPEORM_HOST = host number
TYPEORM_USERNAME = root
TYPEORM_PASSWORD = myPassword
TYPEORM_DATABASE = database
TYPEORM_PORT = rdbms port
TYPEORM_LOGGING = TRUE
```

- dbmate 연결 목적의 DATABASE_URL에는 개인이 로컬 환경에서 어떠한 rdbms를 쓰는지 그 힌트가 될 수 있는 단어와, 포트번호를 지우고 중의적인 단어를 대신 사용해주세요.
- app.js에서 process.env.PORT 를 통해 환경변수로서 저장된 PORT를 변수에 할당하고 사용하고 있다면 .env.sample에도 해당 내용이 들어가야 합니다.
- .env.sample에는 개인이 로컬 환경에서 어떠한 rdbms 쓰는지와 같은 직접적인 그 힌트가 될 수 있는 단어와, 포트번호를 지우고 중의적인 단어를 대신 사용해주세요. 다른 값들도 마찬가지입니다!